lib/generators/factory_bot/model/templates/factories.erb

```diff
@@ -0,0 +1,3 @@
+FactoryBot.define do
+<%= factory_definition.chomp %>
```

Comment:
should this be indented?

in a followup(previous?) PR you use `rstrip` instead of `chomp` should this be using `rstrip`?

Reply:
It shouldn't be indented because `factory_definition` takes care of the indentation. But I agree it looks a bit strange, so I will open a separate PR to fix that.

#306 was merged, so I rebased and switched to `rstrip` 👍